get_gerrit_last_comment_by_id.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "I require jq but it's not installed. Aborting."; exit 1; }
```

Comment:
Please keep readable `if ! command -v jq ...; then ... fi` instead of logical operators and such single lines abominations in scripts. In Python no one wound not write like that, so why we are are doing it in Bash?

Reply:
Deleted one liners. Change done.